habitat-lab/habitat/config/benchmark/rearrange/skills/pick_spot.yaml

```diff
@@ -0,0 +1,69 @@
+# @package _global_
+defaults:
+  - pick
```

Comment:
Thanks Andrew! I have added a detailed explanations yaml config. My thought is that since this is yaml for Spot (as name suggested), we should make sure the yaml contains all the necessary changes for the real-world deployment. I think we still need the base pick skills since some of the measurements/lab sensors are the same. We do not want to have a redundant code. I also leave comments about what is the same and what is the different to the base yaml.